src/main.rs

```diff
@@ -137,11 +246,7 @@ impl EventHandler for MyGame {
 
         let in_game_pos = mint::Point2::<f32>::from([pos.x - x_offset, pos.y - y_offset]);
 
-        if in_game_pos.x >= 0.0
-                && in_game_pos.y >= 0.0
-                && in_game_pos.x <= cell_num_to_coord(self.game_state.width())
-                && in_game_pos.y <= cell_num_to_coord(self.game_state.height()) 
-        {
+        if self.game_zone.in_clickable_zone(pos) {
```

Comment:
<img src="https://github.com/alexesmet/nonogram-gamer-rs/assets/36900282/8d8b072e-982b-47d3-b421-aefb40e8a088" width="200">

Hmm so idea of the clickable zone is that
1. It can draw itself (optionally)??
2. It can tell when it is clicked but we have to handle the click ourselves

Reply:
I just don't have a better idea at that moment but I feel like drawable click zone and invisible click zone must be two different things